index.html

```diff
@@ -33,9 +33,149 @@
+           <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           <div class="tagcontainer">
+            <label>Ruby on Rails</label>
+            <label>Javascript</label>
+            <label>CSS</label>
+            <label>HTML</label>
+           </div>
+           <button class="see-post"><p class="see-button">see project</p></button>
+           </div>
+       
+        <div class="grid-item third">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           <div class="tagcontainer">
+            <label>Ruby on Rails</label>
+            <label>Javascript</label>
+            <label>CSS</label>
+            <label>HTML</label>
+           </div>
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+        <div class="grid-item fourth">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           <div class="tagcontainer">
+            <label>Ruby on Rails</label>
+            <label>Javascript</label>
+            <label>CSS</label>
+            <label>HTML</label>
+           </div>
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+        <div class="grid-item fiveth"><div class="item image"></div> 
+        <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+        <div class="tagcontainer">
+         <label>Ruby on Rails</label>
+         <label>Javascript</label>
+         <label>CSS</label>
+         <label>HTML</label>
+        </div>
+        <button class="see-post"><p class="see-button">see project</p></button></div>
+        <div class="grid-item sixth">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           <div class="tagcontainer">
+            <label>Ruby on Rails</label>
+            <label>Javascript</label>
+            <label>CSS</label>
+            <label>HTML</label>
+           </div>
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+        <div class="grid-item ">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           <div class="tagcontainer">
+            <label>Ruby on Rails</label>
+            <label>Javascript</label>
+            <label>CSS</label>
+            <label>HTML</label>
+           </div>
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+    </div>
+</section>
+<section>
+    <div class="grid-container-another">
+        <div class="about-me">
+            <h2 class="about">About Me</h2>
+            <p class="about-myself"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
```

Comment:
- [ ] your about me is missing the background image which is currently present on the figma template shown below please make sure to add this to your project design :+1: 
![bg-about](https://user-images.githubusercontent.com/89055502/235299711-6f918515-0522-4dc3-8e7e-17f590069850.png)

Reply:
Hi @Efiamotu-1 
I am unable to download the template. when I try to download the template they do not give me it. they give me just an element image. I tried to insert the image in code. I am unable to insert it.
 I appreciate it if you provide any help